Added line item for Reverse number lookup (#3078)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/here-s-what-you-get-with-phone-system.md

|Music on hold | Available in September 2019, Phone System will play default music defined by the service when an external call from the Public Switched Telephone Network (PSTN) is placed on hold.  This feature works for one-to-one PSTN-to-Teams calls in addition to calls made to a call queue. This feature provides on-hold notification parity with other platforms. (All other platforms give a visual cue when a user is put on hold, but the PSTN caller has had no way of knowing.)  On first release, this feature is not configurable by the administrator.| 

> |Reverse Number Lookup | Available in September 2019, Phone System will show called ID for an external phone number, if this number is associated with a user in AAD. If the phone number is not in AAD, then telco provided display name will be shown if available.  

Reviewer: ```suggestion
|Reverse Number Lookup | Available in September, 2019, Phone System will show called ID for an external phone number if the number is associated with a user in AAD. If the phone number is not in AAD, the telco provided display name will be shown if it is available.  
```
